l10n_pt_vat/__openerp__.py

```diff
@@ -21,18 +21,18 @@
 
 {
     "name": "Portugal - IVA",
-    "version": "8.0.0.0.2",
+    "version": "8.0.0.0.3",
     "license": "AGPL-3",
     "depends": ['account', 'base_vat'],
     "author": "Sossia, Odoo Community Association (OCA)",
     "summary": "Portuguese VAT requirements extensions",
-    "website": "http://www.sossia.pt",
-    "category": "Accounting & Finance",
+    "website": "https://github.com/OCA/l10n-portugal",
+    "category": "Localisation/Europe",
```

Comment:
Ok, it'll be done in a minute

-- 
Pedro Castro Silva
Sossia
917213709
A Qui, 19-11-2015 às 07:00 -0800, Daniel Reis escreveu:
> In l10n_pt_vat/__openerp__.py:
> >      "license": "AGPL-3",
> >      "depends": ['account', 'base_vat'],
> >      "author": "Sossia, Odoo Community Association (OCA)",
> >      "summary": "Portuguese VAT requirements extensions",
> > -    "website": "http://www.sossia.pt",
> > -    "category": "Accounting & Finance",
> > +    "website": "https://github.com/OCA/l10n-portugal",
> > +    "category": "Localisation/Europe",
> Nice to know. I would stick to the UK version, with an s, just like
> you were using.
> —
> Reply to this email directly or view it on GitHub.